101-azurefirewall-with-zones-sandbox/azuredeploy.json

```diff
@@ -0,0 +1,397 @@
+      }
+    },
+    "adminUsername": {
+      "type": "string",
+      "metadata": {
+        "description": "Username for the Virtual Machine."
+      }
+    },
+    "adminPassword": {
+      "type": "securestring",
+      "metadata": {
+        "description": "Password for the Virtual Machine."
+      }
+    },
+    "location": {
+      "type": "string",
+      "defaultValue": "[resourceGroup().location]",
+      "metadata": {
+        "description": "Location for all resources."
+      }
+    },
+    "availabilityZones": {
+      "type": "array",
+      "defaultValue": [],
+      "metadata": {
+        "description": "Zone numbers e.g. 1,2,3."
+      }
+    }
+  },
+  "variables": {
+    "vnetAddressPrefix": "10.0.0.0/16",
+    "serversSubnetPrefix": "10.0.2.0/24",
+    "azureFirewallSubnetPrefix": "10.0.1.0/24",
+    "jumpboxSubnetPrefix": "10.0.0.0/24",
+    "nextHopIP": "10.0.1.4",
+    "azureFirewallSubnetName": "AzureFirewallSubnet",
+    "jumpBoxSubnetName": "JumpboxSubnet",
+    "serversSubnetName": "ServersSubnet",
+    "jumpBoxPublicIPAddressName": "JumpHostPublicIP",
+    "jumpBoxNsgName": "JumpHostNSG",
+    "jumpBoxNicName": "JumpHostNic",
+    "jumpBoxSubnetId": "[resourceId('Microsoft.Network/virtualNetworks/subnets', parameters('virtualNetworkName'), variables('jumpBoxSubnetName'))]",
+    "serverNicName": "ServerNic",
+    "serverSubnetId": "[resourceId('Microsoft.Network/virtualNetworks/subnets', parameters('virtualNetworkName'), variables('serversSubnetName'))]",
+    "storageAccountName": "[concat(uniquestring(resourceGroup().id), 'sajumpbox')]",
+    "azfwRouteTableName": "AzfwRouteTable",
+    "firewallName": "firewall1",
+    "publicIPName": "publicIP1",
+    "azureFirewallSubnetId":  "[resourceId('Microsoft.Network/virtualNetworks/subnets',parameters('virtualNetworkName'), variables('azureFirewallSubnetName'))]",
+    "azureFirewallPublicIpId": "[resourceId('Microsoft.Network/publicIPAddresses',variables('publicIPName'))]"
+  },
+  "resources": [
+    {
+      "type": "Microsoft.Storage/storageAccounts",
+      "name": "[variables('storageAccountName')]",
+      "apiVersion": "2017-10-01",
+      "location": "[parameters('location')]",
+      "sku": {
+        "name": "Standard_LRS"
+      },
+      "kind": "Storage",
+      "properties": {}
+    },
+    {
+      "type": "Microsoft.Network/routeTables",
+      "name": "[variables('azfwRouteTableName')]",
+      "apiVersion": "2018-02-01",
+      "location": "[parameters('location')]",
+      "properties": {
+        "disableBgpRoutePropagation": false,
+        "routes": [
+          {
+            "name": "AzfwDefaultRoute",
+            "properties": {
+              "addressPrefix": "0.0.0.0/0",
+              "nextHopType": "VirtualAppliance",
+              "nextHopIpAddress": "[variables('nextHopIP')]"
+            }
+          }
+        ]
+      }
+    },
+    {
+      "name": "[parameters('virtualNetworkName')]",
+      "apiVersion": "2017-06-01",
```

Comment:
update apiVersion

Reply:
Should I update it to latest API version (2019-04-01)? Why does it have to change here? The reason I ask is because we have this in 101-azurefirewall-sandbox/azuredeploy.json and I just took it from there.

Reply:
Done

Reply:
Done